packages/@react-spectrum/s2/style/spectrum-theme.ts

```diff
@@ -670,6 +672,10 @@ export const style = createTheme({
     borderEndWidth: createRenamedProperty('borderInlineEndWidth', borderWidth),
     borderTopWidth: borderWidth,
     borderBottomWidth: borderWidth,
+    borderInlineStartColor: borderColor, // don't know why i can't use renamed
+    borderInlineEndColor: borderColor,
+    borderTopColor: borderColor,
+    borderBottomColor: borderColor,
```

Comment:
Do we really need these? When do the borders have different colors? Adding these will increase the CSS size across all components so want to be sure it's actually needed. Looks like maybe you could set the width of one of the borders to zero instead of setting it to transparent?

Reply:
using 0/1 instead of 1 always with transparent causes elements to shift their position internal to the tree item